### PR TITLE
docs: update existing docs and add new where it's needed

### PR DESCRIPTION
**Pull request by August-Brandt**

### CODE REVIEW

File: docs/dev_docs/frontend_thoughtworks_javascript.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+```
+
+## Thoughtworks one file JS
+The original Thoughtworks tech radar is build on having a Docker container or a Webpack to start a server. Furthermore, the original Thoughtworks js files are split into their own specific "need"-case. In this version of the Tech Radar, all js files are combined into one large script file called <mark style="background-color: #69a8f5; margin:0 4px; padding: 0 4px"> ./src/HTML/js/renderingTechRadar.js </mark>.
+
+## Updating Thoughtworks radar
+Inside renderingTechRadar.js comments outline each block of code, showcasing which Thoughtworks js file the code originated from. To update the Tech Radar properly, locate the commented code block you seek and update them from the Thoughtworks js file.
+
+## Updating Thoughtworks radar with a library or script
+To add a library or another script to the Tech Radar you can add it by using requireJS. First, add you script link or library link inside <mark style="background-color: #69a8f5; margin:0 4px; padding: 0 4px"> ./src/HTML/js/requireConfig.js </mark>.
```
Comment (zqueamish) [inline docs/dev_docs/frontend_thoughtworks_javascript.md:43]:
```suggestion
To add a library or another script to the Tech Radar you can add it by using requireJS. First, add your script link or library link inside <mark style="background-color: #69a8f5; margin:0 4px; padding: 0 4px"> ./src/HTML/js/requireConfig.js </mark>.
```

File: docs/user_docs/quickstart.md [hunk 2]
```diff
@@ -1,6 +1,40 @@
 # Welcome to Tech_Radar quickstart guide
-In this guide you will find the essential commands for the program. The program is split into 3 primary commands: Fetcher, Merger, and Generator.
+In this guide you will see how to install and setup the program as well as find the essential commands for the program. The program is split into 3 primary commands: Fetcher, Merger, and Generator.
 
+# Installation and setup
+To install the program: 
+- Go to the latest release on the repository and download the version that fits your operating system.
```
Comment (zqueamish) [inline docs/user_docs/quickstart.md:6]:
```suggestion
- Go to the latest release on the [repository](https://github.com/NovoNordisk-OpenSource/decentralized-tech-radar/releases) for this program, and download the version that fits your operating system. 
```

**Review by zqueamish — CHANGES_REQUESTED** [review]

**Review by JonasSkjodt — APPROVED** [review]

**Review by Alex-itu — APPROVED** [review]
Looks good to me, with or without the changes.

File: docs/user_docs/quickstart.md [hunk 3]
```diff
@@ -22,25 +70,53 @@ Example:
...
+
+
 `path/to/csvfile` being a path to a locally stored csvfile with correct format *__see spec_file_format.md__*  
 
+
 Example:  
 ![Gif of merging using the CLI](https://github.com/NovoNordisk-OpenSource/decentralized-tech-radar/blob/main/docs/images/quickstart_gifs/Merging.gif)
 
-The merger also includes a cache flag this will run the merger using the cache folder generated from the fetcher (If the cache is located next to the program):
+The merger also includes a cache flag. This will merge all files located in the cache folder:
```
[the remark this answers is not in the document]
Comment (August-Brandt) [reply docs/user_docs/quickstart.md:92]: Technically the merge has no idea if the files are from earlier fetching. It just merges all files in the cache folder

**Review by Slug-Boi — APPROVED** [review]

**Pull request merged**